client/src/components/RelatedItems/OutfitCreation.jsx

```diff
@@ -22,11 +28,31 @@ const OutfitCreation = ({ productId, calcRating, saleAndImageSetter, renderPrice
             return [...currOutfits, product.data];
```

Comment:
what's currOutfits?

Reply:
currOutfits are the currently stored outfits. I'm using the function mode for useState to make sure everything gets rendered when I want it to. The function mode takes in an argument that's the current state.